axelrod/tests/unit/test_match_generator.py

```diff
@@ -125,7 +125,7 @@ def setUpClass(cls):
     @given(repetitions=integers(min_value=1, max_value=test_repetitions),
            prob_end=floats(min_value=0, max_value=1))
     @example(repetitions=test_repetitions, prob_end=.5)
-    def test_build_match_chunks(self, repetitions, prob_end):
+    def _chunks(self, repetitions, prob_end):
```

Comment:
If this has got the same name as the other one it shouldn't as they actually overwrite so rename this to something different...